dashboard/app/controllers/test_status_controller.rb

```diff
@@ -0,0 +1,33 @@
+require 'aws-sdk'
+require 'date'
+
+# API that serves UI test status based on uploaded test run S3 logs and their
+# metadata.  See runner.rb, test_status.haml and test_status.js for more
+# information.
+class TestStatusController < ApplicationController
```

Comment:
We might want to follow our [new API pattern](https://docs.google.com/document/d/10FKEASCjoANN2wLTIVSRM6EjPUqDj-VnVY1CM8LQll0) and put this under `/api/v1/`.